modules/virt-additional-scc-for-kubevirt-controller.adoc

```diff
@@ -8,11 +8,9 @@
 
 Security context constraints (SCCs) control permissions for pods. These permissions include actions that a pod, a collection of containers, can perform and what resources it can access. You can use SCCs to define a set of conditions that a pod must run with to be accepted into the system.
 
-The `kubevirt-controller` is a cluster controller that creates the virt-launcher pods for virtual machines in the cluster. These virt-launcher pods are granted permissions by the `kubevirt-controller` service account.
```

Comment:
What's wrong with this line?

Reply:
It seemed too repetitive (very similar to line 13 but slightly less informative)

Reply:
I think I'll remove the heading above line 13 so that there is no separation. :)

Reply:
This is about the `virt-controller` (not `kubevirt-controller`, probably a typo) Kubernetes controller (https://kubernetes.io/docs/concepts/architecture/controller/) installed by KubeVirt.

The lines below are about the `kubevirt-controller` service account (https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/), which is a completely different Kubernetes object.

I see value in explaining both...

Reply:
Oh, good to know. I will add it back!

Reply:
My apologies. 😅